pyrasterframes/src/main/python/docs/reference.pymd

```diff
@@ -517,33 +506,20 @@ Computes a statistical summary of cell values within each row of `tile`. Resulti
 
 These functions compute statistical summaries over all of the cell values *and* across all the rows in the DataFrame or group. Example use below computes a single double-valued mean per month, across all data cells in the `red_band` `tile` type column. This would return at most twelve rows.
 
-
```

Comment:
I think now the discussion above about the monthly mean should also be removed. As it's unclear what it is talking about.